scipy-1.0/paper.tex

```diff
@@ -54,7 +54,24 @@ \subsection*{Submodule organisation}
 
 \subsection*{Common infrastructure}
 
-\subsection*{Language preferences}
+\subsection*{Language choices}
+Python / Cython, Fortran, C and C++ are the programming languages
+used to implement scientific algorithms in the SciPy library. An analysis
+of our code base using the \texttt{linguist} library provides a detailed
```

Comment:
It is also written on top of the GitHub repo (click the color bar), where I got the numbers from.